nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -321,10 +322,13 @@
     // analytics defaults
     public static final String DEFAULT_ANALYTICS_PREDICTION_ENABLED = "false";
     public static final String DEFAULT_ANALYTICS_PREDICTION_INTERVAL = "3 mins";
-    public static final String DEFAULT_ANALYTICS_QUERY_INTERVAL = "3 mins";
+    public static final String DEFAULT_ANALYTICS_QUERY_INTERVAL = "5 mins";
     public final static String DEFAULT_ANALYTICS_CONNECTION_MODEL_IMPLEMENTATION = "org.apache.nifi.controller.status.analytics.models.OrdinaryLeastSquares";
     public static final String DEFAULT_ANALYTICS_CONNECTION_SCORE_NAME = "rSquared";
     public static final double DEFAULT_ANALYTICS_CONNECTION_SCORE_THRESHOLD = .90;
+    // This is the time, below which the status history graph will actively display
+    // backpressure estimates if prediction analytics are enabled.
+    public static final String DEFAULT_ANALYTICS_STATUS_GRAPH_THRESHOLD = "6 hrs";
```

Comment:
Is there a rationale for using 6 hours?

Reply:
It was a value chosen at the suggestion of someone familiar with the project. Beyond that, no. It can be set to whatever the user desires, but if it gets too large then the graph can get difficult to read.